lhotse/cut.py

```diff
@@ -172,6 +192,9 @@ def pad(self, desired_duration: Seconds) -> AnyCut:
         :param desired_duration: The cut's minimal duration after padding.
         :return: a padded MixedCut if desired_duration is greater than this cut's duration, otherwise self.
         """
+        if self.features is None:
```

Comment:
I think this `if` is unnecessary - `PaddingCut` already supports padding audio with zeros IIRC.

Reply:
That would actually make a great unit test.